libs/services/designer-client-services/src/lib/standard/manifests/xml.ts

```diff
@@ -0,0 +1,210 @@
+import { SettingScope } from '@microsoft/utils-logic-apps';
+import type { OperationManifest } from '@microsoft/utils-logic-apps';
+
+const connector: any = {
+  id: 'connectionProviders/xmlOperations',
+  name: 'xmlOperations',
+  properties: {
+    description: 'XML Operations',
+    displayName: 'XML Operations',
+  },
+};
+
+const settings: any = {
+  secureData: {},
+  trackedProperties: {
+    scopes: [SettingScope.Action],
+  },
+};
+
+export const xmlTransformManifest = {
+  properties: {
+    iconUri: 'https://logicappsv2resources.blob.core.windows.net/icons/xml.svg',
+    brandColor: '#804998',
+    summary: 'Transform XML',
+    description: 'Transform XML using XSLT map.',
+
+    inputs: {
+      type: 'object',
+      required: ['content', 'mapName', 'xsltParameters'],
+      properties: {
+        containerId: {
+          name: 'containerId',
+          title: 'Container Id',
+          summary: 'Function container',
+          description: 'The function container',
+          type: 'string',
+          visibility: 'internal',
+          'x-ms-dynamic-values': {
```

Comment:
x-ms-dynamic-values are not in manifest they are swagger based dynamic extension, we should be using x-ms-dynamic-list.

Reply:
Where should I get `x-ms-dynamic-list` property information from?  I don't see any mention of it in BPMUX.
Should I also just remove the `x-ms-dynamic-values`?  From your comment I thought the two props used the same information but they don't really match up.

Reply:
lets leave the manifest updates in this PR, we will work on it separately since we have to make sure this manifest works for the runtime and authoring.